src/Storages/Kafka/StorageKafka.cpp

```diff
@@ -519,13 +522,14 @@ void registerStorageKafka(StorageFactory & factory)
         CHECK_KAFKA_STORAGE_ARGUMENT(1, kafka_broker_list)
         CHECK_KAFKA_STORAGE_ARGUMENT(2, kafka_topic_list)
         CHECK_KAFKA_STORAGE_ARGUMENT(3, kafka_group_name)
-        CHECK_KAFKA_STORAGE_ARGUMENT(4, kafka_format)
-        CHECK_KAFKA_STORAGE_ARGUMENT(5, kafka_row_delimiter)
-        CHECK_KAFKA_STORAGE_ARGUMENT(6, kafka_schema)
-        CHECK_KAFKA_STORAGE_ARGUMENT(7, kafka_num_consumers)
-        CHECK_KAFKA_STORAGE_ARGUMENT(8, kafka_max_block_size)
-        CHECK_KAFKA_STORAGE_ARGUMENT(9, kafka_skip_broken_messages)
-        CHECK_KAFKA_STORAGE_ARGUMENT(10, kafka_commit_every_batch)
+        CHECK_KAFKA_STORAGE_ARGUMENT(4, kafka_client_id)
```

Comment:
that is bad idea. we don't want to support positional parameter, and for positional paramters it should be added to to end.

Reply:
That brakes cackward compatibility

Reply:
done.